### PR TITLE
Enable branch cache by default

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -95,13 +94,10 @@ func main() {
...
-		if branchCacheFolder == "" {
-			branchCacheFolder, err = server.ReadBranchCacheFolder(
-				ctx, branchCacheBucket, branchCacheVersionFile)
-			if err != nil {
-				log.Fatalf("Failed to read branch cache folder: %v", err)
-			}
+		branchCacheFolder, err := server.ReadBranchCacheFolder(
+			ctx, branchCacheBucket, branchCacheVersionFile)
+		if err != nil {
+			log.Fatalf("Failed to read branch cache folder: %v", err)
```
Comment (pradh) [inline cmd/main.go:100]: Should this be fatal or more graceful?

**Review by pradh — APPROVED** [review]
Thanks for the change!

File: cmd/main.go (hunk 1 above)
Comment (shifucun) [reply cmd/main.go:100]: there is no monitoring on errors yet, for now can make it fatal so we know the branch cache load has issue?

Comment (pradh) [reply cmd/main.go:100]: ok. and if bad things happen we just set enableBranchCache to false...

Comment (shifucun) [reply cmd/main.go:100]: sg

**Pull request merged**